tests/tests_integration/test_api/test_data_modeling/test_spaces.py

```diff
@@ -10,8 +10,10 @@
 from cognite.client.exceptions import CogniteAPIError
 
 
-@pytest.fixture(scope="function")
-def cdf_spaces(cognite_client: CogniteClient) -> SpaceList:
+@pytest.fixture(scope="session")
+def cdf_spaces(cognite_client: CogniteClient, integration_test_space: Space) -> SpaceList:
+    # The integration test space is created in the fixture integration_test_space.
+    # This ensures that at least one space exists in CDF.
```

Comment:
I prefer to use `usefixtures` to indicate fixture dependency when not directly needed:

```suggestion
@pytest.fixture(scope="session")
@pytest.mark.usefixtures("integration_test_space")
def cdf_spaces(cognite_client: CogniteClient) -> SpaceList:
```

Reply:
I learned something new, did not know about the usefixture fixture.